azurerm/internal/services/healthcare/tests/resource_arm_healthcare_service_test.go

```diff
@@ -146,7 +149,7 @@ resource "azurerm_healthcare_service" "test" {
     "${data.azurerm_client_config.current.service_principal_object_id}",
   ]
 }
-`, data.RandomInteger, location, data.RandomInteger)
+`, data.RandomInteger, location, data.RandomInteger/10) //name can only be 24 chars long
```

Comment:
could we use RandomString here instead:

```suggestion
`, data.RandomInteger, location, data.RandomString)
```

Reply:
I don't see the benefit from using random string here? this works and retains the date information from random int